src/convertToObject.js

```diff
@@ -3,16 +3,63 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+function splitKey(string) {
+  const firstIndex = string.lastIndexOf('\n') + 1;
+  const lastIndex = string.indexOf(':');
+
+  return string.slice(firstIndex, lastIndex).trim();
+}
+
+function splitValue(string) {
+  const valuePart = string.slice(string.indexOf(':') + 1);
+
+  return valuePart.replace(/\s+/g, ' ').trim();
+}
```

Comment:
try to use the split method by :. First element will be key, second value

Reply:
With the initial split (';') we get hyphens (/n) at the beginning of the line, and if we do another split (as I tried), then these /n get into the keys and we still have to get rid of them.